src/pages/Main/Main.js

```diff
@@ -1,8 +1,282 @@
+                  />
+                  <h3>선릉 맛집 베스트 30곳</h3>
+                  <h4>"구석구석 맛집으로 가득한 이 곳!"</h4>
+                </li>
+                <li>
+                  <img
+                    alt="테마이미지"
+                    src="https://mp-seoul-image-production-s3.mangoplate.com/keyword_search/meta/pictures/0zjqwsskbtpucd0i.jpg?fit=around|600:400&crop=600:400;*,*&output-format=jpg&output-quality=80"
+                  />
+                  <h3>수제버거 맛집 베스트 15곳</h3>
+                  <h4>"부실한 패스트푸드는 이제 그만!"</h4>
+                </li>
+                <li>
+                  <img
+                    alt="테마이미지"
+                    src="https://mp-seoul-image-production-s3.mangoplate.com/keyword_search/meta/pictures/vh3szktueppvzbam.png?fit=around|600:400&crop=600:400;*,*&output-format=jpg&output-quality=80"
+                  />
+                  <h3>파전 맛집 베스트 3곳</h3>
+                  <h4>"비 오는 날은 파전에 막걸리지"</h4>
+                </li>
+                <li>
+                  <img
+                    alt="테마이미지"
+                    src="https://mp-seoul-image-production-s3.mangoplate.com/keyword_search/meta/pictures/4y8ry99myorgriw9.jpg?fit=around|600:400&crop=600:400;*,*&output-format=jpg&output-quality=80"
+                  />
+                  <h3>선릉 맛집 베스트 30곳</h3>
+                  <h4>"구석구석 맛집으로 가득한 이 곳!"</h4>
+                </li>
+                <li>
+                  <img
+                    alt="테마이미지"
+                    src="https://mp-seoul-image-production-s3.mangoplate.com/keyword_search/meta/pictures/0zjqwsskbtpucd0i.jpg?fit=around|600:400&crop=600:400;*,*&output-format=jpg&output-quality=80"
+                  />
+                  <h3>수제버거 맛집 베스트 15곳</h3>
+                  <h4>"부실한 패스트푸드는 이제 그만!"</h4>
+                </li>
+              </div>
+            </ul>
+            <button
+              className={slideNum < totalSlide ? 'arrowBtnRight' : 'none'}
+              onClick={handleBtn}
+            >
+              <i class="fas fa-chevron-right fa-3x" id="arrowRight"></i>
+            </button>
+            <ul className="slickDots">
+              <li className="slickActive">
+                <i
+                  class="fas fa-circle fa-xs"
+                  id={slideNum === 1 ? 'colorDot' : ''}
+                  index={1}
+                  onClick={handleDotBtn}
+                ></i>
+              </li>
+              <li className="slickActive">
+                <i
+                  class="fas fa-circle fa-xs"
+                  id={slideNum === 2 ? 'colorDot' : ''}
+                  index={2}
+                  onClick={handleDotBtn}
+                ></i>
+              </li>
+              <li className="slickActive">
+                <i
+                  class="fas fa-circle fa-xs"
+                  id={slideNum === 3 ? 'colorDot' : ''}
+                  index={3}
+                  onClick={handleDotBtn}
+                ></i>
+              </li>
+            </ul>
```

Comment:
```suggestion
            <ul className="slickDots">
              <Dot index={1} isSelected={slideNum === 1} onClick={handleDotBtn} />
              <Dot index={2} isSelected={slideNum === 2} onClick={handleDotBtn} />
              <Dot index={3} isSelected={slideNum === 3} onClick={handleDotBtn} />
            </ul>
```
이런 식으로 작성할 수 있다면 조금 더 render 함수 안의 내용들이 어떤 의미를 가지고 있는지 직관적으로 알 수 있을 것 같습니다.